--- lingot-0.6.1/Makefile.in~	2005-07-25 01:41:26.000000000 +0200
+++ lingot-0.6.1/Makefile.in	2006-01-22 21:36:30.000000000 +0100
@@ -42,9 +42,8 @@
 	$(CXX) $(C_FLAGS) -c $< -o $@
 
 install: $(PROJECT)
-	install -d @prefix@
-	install -d @prefix@/bin
-	install $(PROJECT) @prefix@/bin
+	install -d $(DESTDIR)@prefix@/bin
+	install $(PROJECT) $(DESTDIR)@prefix@/bin
 
 clean:
 	find -type f -path '*.o' | xargs rm -f
